src/vaults/vaults.v3.controller.ts

```diff
@@ -100,4 +101,30 @@ export class VaultsV3Controller {
       timestamps.split(',').map((n) => Number(n)),
     );
   }
+
+  @Hidden()
+  @UseCache()
+  @Get('/inrange/snapshots')
+  @ContentType('json')
+  async getVaultSnapshotsInRange(
+    @QueryParams('vaultAddr') vaultAddr: string,
```

Comment:
let's use `address` 
we aren't having v3 charts in production yet fully so its ok if we drop it for a bit.